actor/builtin/storagemarket/storagemarket_test.go

```diff
@@ -234,3 +299,52 @@ func deriveMinerAddress(creator address.Address, nonce uint64) (address.Address,
 
 	return address.NewActorAddress(buf.Bytes())
 }
+
+// mustCreateStorageMiner creates a storage miner for the given state tree & storage map
+// and returns the address of the created miner.
+func mustCreateStorageMiner(t *testing.T, st state.Tree, vms vm.StorageMap, height uint64) address.Address {
```

Comment:
Please consider building on top of existing work in testhelpers/core.go to consolidate our testing framework.  There are already `CreateTestMiner` and `CreateTestMinerWith`.  Neither is exactly what you want because you need to specify height but it should be straightforward to add this param to `CreateTestMinerWith` and then update callers.